### PR TITLE
[rllib] Include config dicts in the sphinx docs

### PR DESCRIPTION
**Pull request by ericl**
## What do these changes do?

Inlines the rllib configs into the sphinx docs using the literal inline directive.

### CODE REVIEW

File: doc/source/rllib-algorithms.rst [hunk 1]
```diff
@@ -73,6 +80,13 @@ SpaceInvaders  843                              ~300
 
    IMPALA solves Atari several times faster than A2C / A3C, with similar sample efficiency. Here IMPALA scales from 16 to 128 workers to solve PongNoFrameskip-v4 in ~8 minutes.
 
+**IMPALA-specific configs**:
```
Comment (richardliaw) [inline doc/source/rllib-algorithms.rst]: Maybe mention the default configs/`with_common_config` on this page
screenshot 2018-10-16 11 37 10

Reference to `with_common_config` is not on this page but on the `training` which can be pretty confusing...

**Review by ericl — COMMENTED** [review]
Addressed

**Review by richardliaw — APPROVED** [review]

**Comment by richardliaw** [comment]
For Tune, are you referring to the automatically filled fields?

**Comment by ericl** [comment]
@richardliaw Yeah I recall we have that list of optional / filled fields.

**Comment by richardliaw** [comment]
Yeah we could do that

**Pull request merged**